Gemfile

```diff
@@ -1,22 +1,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
```

Comment:
I don't think you want to. This is purely for testing gem compatibility, aka the build, I would just leave it as is. The .gemspec is published to other gems.